### ISSUE TITLE
Why don't you guys add all the stuff of proot from start?

### ISSUE THREAD

**Issue by sayantan300** [termux/termux-packages#7315]
If you guys could add all the folders and etc stuff to termux, then it could save a lot of useless storage, because we would be installing a lot of same packages in termux and in distro, like git, so we would be wasting double storage...

**Comment by agnostic-apollo** [comment]
Because `proot` currently is not fully stable on all devices and has other issues so its not used by default.

https://github.com/termux/proot/issues

https://www.reddit.com/r/termux/comments/ho16du/eli5_what_are_the_limitations_and_problems_of/fxeyuu3/

If you just use packages in `proot`, then don't install extra packages like `git` in termux that are [not part of bootstrap](https://github.com/termux/termux-packages/blob/master/scripts/generate-bootstraps.sh#L316). Termux F-Droid Install+bootstrap installation size is `~180MB` ([`~120MB` for debug](https://github.com/termux/termux-app/releases/tag/v0.117)), for which users already complain about is too large. Adding `proot` stuff on top of that would make it worse and not everybody uses `proot` anyways (including me).

**Comment by xeffyr** [comment]
If you need proot-based terminal, use https://github.com/CypherpunkArmory/UserLAnd instead.
